CONTRIBUTING.md

```diff
@@ -39,14 +39,46 @@ mvn verify
 ```
 
 After a successful build the RAPIDS Accelerator jar will be in the `dist/target/` directory.
+This will build the plugin for a single version of Spark.  By default this is Apache Spark
+3.0.1. To build against other versions of Spark you use the `-Dbuildver=XXX` command line option
+to maven. For instance to build Spark 3.1.1 you would use:
 
-### Building Shims for Spark Snapshot Versions
+```shell script
+mvn -Dbuildver=311 verify
+```
+You can find all available build versions in the top level pom.xml file. Currently
+they include 301, 302, 303, 304, 311, 312, 313, 320 and 311cdh. If you are building
+Databricks suggest you use the `jenkins/databricks/build.sh` script and modify it for
+the version you want.
+
+To get an uber jar with more then 1 version you have to `mvn install` each version
+and then use one of the defined profiles in the dist module. See the next section
+for more details.
+
+### Building a distribution for multiple versions of Spark
+
+By default the distribution jar only includes code for a single version of Spark. If you want
+to create a jar with multiple versions we currently have 4 options.
 
-By default the build will only include shims for released versions of Spark. To include shims
-for snapshot versions of Spark still under development, use the `snapshot-shims` Maven profile
-(e.g.: add `-Psnapshot-shims` to the Maven command-line). Note that when a snapshot Spark version
-later becomes an official release, the snapshot shim for that version may no longer build due to
-missing snapshot artifacts for that Spark version.
+1. Build for all Apache Spark versions and CDH with no SNAPSHOT versions of Spark, only released. Use `-Pallnosnapshots`.
```

Comment:
suggestions on naming conventions here?  I realize its a bit weird with allno or all nowithDB, etc..

Reply:
It cannot be all if it does not include databricks.  I would replace all with default, standard, or just drop it all together.

`-PnoSnapshots`
`-Psnapshots`
`-PnoSnaphsotsWithDatabricks`
`-PsnapshotsWithDatabricks`

Also we should have a way to distinguish words in the profiles. snake_case is good, but I don't know what convention (if any) maven has or that we have used in our poms.